media-sound/libopenmpt/libopenmpt-0.2.7386~beta20.3.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="OpenMPT based module player library and command-line player"
+DESCRIPTION="libopenmpt is a cross platform C++ and C libary to decode tracked \
+music files (modules) int a raw PCM audio stream. openmpt123 is a \
+cross-platform command line or terminal based module file player."
+HOMEPAGE="https://lib.openmpt.org/libopenmpt/"
+COPYRIGHT="2004-2016 OpenMPT contributors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${portVersion/\~/-}-autotools.tar.gz"
+CHECKSUM_SHA256="a6a7e6da1ae66e1cf46985ee92c182e50652d71b96135e9fa6048e132d844753"
+SOURCE_DIR="libopenmpt-${portVersion%~*}-autotools"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]; then
+	commandBinDir=$binDir
+else
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	libopenmpt$secondaryArchSuffix = $portVersion
+	lib:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
+	cmd:openmpt123 = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libmpg123$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libltdl$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libFLAC$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libopenmpt${secondaryArchSuffix}_devel
+	devel:libopenmpt$secondaryArchSuffix = 0.0.20 compat >= 0
+	"
+REQUIRES_devel="
+	libopenmpt$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	devel:libmpg123$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libltdl$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libFLAC$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:autoheader
+	cmd:automake
```

Comment:
I have 2 questions regarding the REQUIRES and BUILD_REQUIRES:
1) When I sort and merge the outputs of
```
objdump -x /lib/x86/libopenmpt.so.0.0.20  | grep NEEDED
```
or
```
objdump -x /bin/openmpt123 | grep NEEDED 
```
I find that the current REQUIRES (and BUILD_REQUIRES) are only missing one lib: lib:libvorbisfile. But this one is provided by the same package which provides lib:libvorbis.
So my first question is: should'nt we be also adding a lib:libvorbisfile$secondaryArchSuffix to REQUIRES and a devel:libvorbisfile$secondaryArchSuffix to BUILD_REQUIRES ?

2)  I noticed that the order of these lines in REQUIRES (and BUILD_REQUIRES) is the same as the order of these libs when we use the objdump command.
So my second question is: is it OK to keep the current order? Or shouldn't we sort them alphabetically, like we (try) to do for all our recipes?

I guess the right answers are: "add the libvorbisfile" & "sort alphabetically".
@korli, am I correct?
Thanks!